lib/features/popup-menu/PopupMenu.js

```diff
@@ -16,6 +16,11 @@ import {
 
 var DATA_REF = 'data-id';
 
+var CLOSE_EVENTS = [
+  'contextPad.close',
```

Comment:
I left it but am not sure if we should really depend on the Context Pad here. Popup Menu can function correctly without that component.

Reply:
Actually, in the case of `contextPad.close` I'd rather see that behaviour in the `bpmn-js` as this is the place where PopupMenu is coupled with Context Pad.

Reply:
I agree. This should not belong to diagram-js. The popup menu has no idea that the context pad even exists.

Reply:
On the other hand: We had that quirky behavior in here. So if there is no good place to put it in bpmn-js we should simply keep it here.

Reply:
The good place I could think of would be the Replace Menu which is a popup that closes with the Context Pad being closed. Anyway, this is outside of scope of this PR.